kai/reactive_codeplanner/task_runner/analyzer_lsp/api.py

```diff
@@ -27,7 +27,7 @@ def __str__(self) -> str:
         else:
             shadowed_priority = self.__class__.priority
 
-        return f"{self.__class__.__name__}<loc={self.file}:{self.line}:{self.column}, message={self.violation.description}>(priority={self.priority}({shadowed_priority}), depth={self.depth}, retries={self.retry_count})"
+        return f"{self.__class__.__name__}<loc={self.file}:{self.line}:{self.column}, message={self.violation.id}>(priority={self.priority}({shadowed_priority}), depth={self.depth}, retries={self.retry_count})"
```

Comment:
Let's change `message=` to something like `violation.id=`

Reply:
fixed